src/main/java/edu/hawaii/its/api/service/UpdateMemberService.java

```diff
@@ -350,31 +351,39 @@ private GroupingAddResults addOwners(String groupingPath, List<String> uhIdentif
         AddMembersResults addMembersResults =
                 grouperApiService.addMembers(groupingPath + GroupType.OWNERS.value(), uhIdentifiers);
         GroupingAddResults results = new GroupingAddResults(addMembersResults);
-        timestampService.update(results);
+        List<String> groupPaths = new ArrayList<>();
+        groupPaths.add(groupingPath);
+        timestampService.update(results, groupPaths);
         return results;
     }
 
     private GroupingAddResult addOwner(String groupingPath, String uhIdentifier) {
         AddMemberResult addMemberResult =
                 grouperApiService.addMember(groupingPath + GroupType.OWNERS.value(), uhIdentifier);
         GroupingAddResult result = new GroupingAddResult(addMemberResult);
-        timestampService.update(result);
+        List<String> groupPaths = new ArrayList<>();
+        groupPaths.add(groupingPath);
+        timestampService.update(result, groupPaths);
```

Comment:
I think these groupPaths lists created here can be moved into deeper into the timestampService.update() function. I would rename the timestampService.update() function to something like updateAll (you can think of something better) which only takes in the result as a param rather than a result param and list param. Then that new function can handle creating a list of groupPaths to update.